app/controllers/main_controller.rb

```diff
@@ -18,8 +18,9 @@ def add_status
 
     message = params[:message]
     author = params[:author]
+    media = params[:media]
 
-    s = Status.new(message: message, author: author)
+    s = Status.new(message: message, author: author, media: media)
```

Comment:
i think this is all it takes? looks like `Status` is basically just a wrapper over a generic record structure?

Reply:
So status is a wrapper over a database object! 

Here's the "model" [code](https://github.com/TessHuelskamp/apartment-website/blob/main/app/models/status.rb) where you can define functions on top of the field. And the original [migration](https://github.com/TessHuelskamp/apartment-website/blob/main/db/migrate/20220711024015_create_statuses.rb) I added. Instructions to generate a migration to add a `text` media url are over [here](https://guides.rubyonrails.org/v3.2/migrations.html).

Normally, you'd add a migration and _then_ the code changes, but given that this project is Just For Fun ™️, feel free to add the migration and code change in one step (I'll do the db migration and deploy once this lands bc I haven't set up CI on this lovely site)

Reply:
@TessHuelskamp ahaaaa got it the migration is what i was missing - i saw the model code extending/implementing `ApplicationRecord` and just assumed that was a way to define arb business logic on generic json objects

will add a db migration then merge (also switching up author display on msgs from first char to full string in case you didnt see)

Reply:
Ahh yeah - usually the models do a little but more than just store data.  E.g., we could do something like this if we want to get fancy with the author:

```ruby
def author_display_name
  return author&.chr 
end
```

Author changes look good!